package.json

```diff
@@ -7,7 +7,7 @@
     "@azure-tools/typespec-azure-portal-core": "0.45.0",
     "@azure-tools/typespec-azure-resource-manager": "0.45.0",
     "@azure-tools/typespec-azure-rulesets": "0.45.0",
-    "@azure-tools/typespec-client-generator-cli": "0.9.4",
+    "@azure-tools/typespec-client-generator-cli": "0.11.1",
```

Comment:
Let's hold off until 0.11.2 since we're resolving some issues in tsp-client.

Reply:
Don't worry, this can't be merged until tests are passing.